src/Applications/LDAS_App/ldas_setup

```diff
@@ -1316,13 +1334,49 @@ def _printExeInputKeys(rqdExeInpKeys):
     print '# Format for start/end times is yyyymmdd hhmmss.           #'
     print '#                                                          #'
     print '############################################################'
-    print
+    print 
     print 'EXP_ID:'
     print 'EXP_DOMAIN:'
     print 'NUM_LDAS_ENSEMBLE:'
     print 'BEG_DATE:'
     print 'END_DATE:'
-    
+
+    print
+    print '############################################################'
+    print '#                                                          #'
+    print '#                LADAS COUPLING EXP INFO                   #'
+    print '#                                                          #'
+    print '#    "LADAS_COUPLING" option:                              #'
+    print '#                1 -- ldas coupled with adas ${ADAS_exp}   #'
+    print '#                0 -- ldas not coupled with adas           #'
+    print '#                                                          #'
+    print '#  Requirements for LADAS_COUPLING = 1:                    #'
+    print '#                                                          #'
+    print '#  (1) BEG_DATE must be consistent with ${ADAS_exp}        #'
+    print '#                first cycle date(21z, 3z, 9z or 15z)      #'
+    print '#                                                          #'        
+    print '#  (2) EXP_DOMAIN must be global CS grid as in ${ADAS_exp} #'
+    print '#                                                          #'
+    print '#  (3) MET_TAG  must be set to  ${ADAS_exp}__Nx+-          #'
+    print '#      MET_PATH must be set to  ${ADAS_exp}/lana/forc      #' 
+    print '#    After ldas exp setup, verify the following link:      #'        
+    print '#    ../input/met_forcing/forc -> ${ADAS_exp}/lana/forc    #'
+    print '#                                                          #'
+    print '#  (4) BCS_PATH must be consistent with that of            #'
+    print '#      ${ADAS_exp}/run/lnbcs                               #'
+    print '#                                                          #'
+    print '#  (5) JOB_SGMT must be set to 6h (ADAS analysis window)   #'
+    print '#                                                          #'
+    print '#  (6) NUM_SGMT must be set to 1                           #'
+    print '#                                                          #'
+    print '#  (7) instantaneous "catch_progn_incr" must be in HIST    #'
+    print '#      collection; time step must match that of LDAS       #'
+    print '#      analysis                                            #'
+    print '#                                                          #'
+    print '############################################################'
+    print
+    print 'LADAS_COUPLING:'
```

Comment:
Shouldn't we put the default "0" here?  It looks like in ~line 308 the 0 is wired into python.

Reply:
yes it should have default 0 in the print line here.

Reply:
I added the default of '0'